research/datasets.py

```diff
@@ -1,19 +1,66 @@
 from functools import partial
 from typing import Literal, Optional
+from dataclasses import dataclass
+import copy
 
 import torch
 from torch.utils.data import DataLoader
 
 from lizrd.text import datasets, packers, data, tokenizers
+from lizrd.support.misc import get_ith_chunk
+
+
+@dataclass
+class BatchSizeRampupConfig:
```

Comment:
can we have some assert that will make sure that this class stores 2 lists of ints of the same length?

Reply:
We have this assert in check_args. I am not sure if it makes sense to add it also here

Reply:
it would be for code readability.

Reply:
I'd also add 1 line description for the class, about what is it for, since config is very widely used sting in this repo

Reply:
ok, adding